docs/spectrumeditorwidget.rst

```diff
@@ -7,6 +10,7 @@ The **Spectrum Editor Widget** is a control to add, remove, and edit spectra.
 
 .. figure:: _images/spectrum-editor-widget.png
    :alt: Spectrum editor widget.
+   :width: 75%
```

Comment:
Earlier: change spectra to spectrums.
A Spectrum is an object which describes how numerical values (of components of a graphics' *Data field*) are converted into red, green, blue and alpha colour components on the display.